src/features/status-bar/components/StatusBar.jsx

```
@@ -84,6 +89,8 @@ const JobStatusBar = () => {
   const [output, setMessage] = useState('');
   const { updateJob, dismissJob } = useJobsStore();
   const setActiveMapCategory = useSetActiveMapCategory();
+  const setMapLayerParameters = useMapStore((state) => state.setMapLayerParameters);
```

ESLint

[error] Replace `(state)·=>·state.setMapLayerParameters` with `⏎····(state)·=>·state.setMapLayerParameters,⏎··`

(prettier/prettier)